Mathlib/Analysis/Analytic/IteratedFDeriv.lean

```diff
@@ -207,7 +207,7 @@ theorem HasFPowerSeriesOnBall.iteratedFDeriv_eq_sum
     (h : HasFPowerSeriesOnBall f p x r) (h' : AnalyticOn 𝕜 f univ) {n : ℕ} (v : Fin n → E) :
     iteratedFDeriv 𝕜 n f x v = ∑ σ : Perm (Fin n), p n (fun i ↦ v (σ i)) := by
   simp only [← iteratedFDerivWithin_univ, ← hasFPowerSeriesWithinOnBall_univ] at h ⊢
-  exact h.iteratedFDerivWithin_eq_sum h' uniqueDiffOn_univ (mem_univ x) v
+  exact h.iteratedFDerivWithin_eq_sum h' .univ (mem_univ x) v
```

Comment:
I am not a big fan of such changes: sure, Lean knows that it expects a `UniqueDiffOn` here, so it knows that `.univ` has to mean `UniqueDiffOn.univ`. However, if a reader wants to understand what is going on, he has to check the type or the expected type, which takes some time, so to me this is strictly worse than `UniqueDiffOn.univ` which doesn't require any effort from the reader. What do you think?

Reply:
Then we don't need to rename anything. Let me split the new simp attributes into a new pull request, then we can drop this one or discuss naming convention later.